metricbeat/module/coredns/stats/_meta/data_request_size_bytes_event.json

```diff
@@ -0,0 +1,78 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "coredns": {
+        "stats": {
+            "dns": {
+                "request": {
+                    "size": {
+                        "bytes": {
+                            "bucket": {
+                                "+Inf": 2210,
+                                "0": 0,
+                                "100": 2210,
+                                "1023": 2210,
+                                "16000": 2210,
+                                "200": 2210,
+                                "2047": 2210,
+                                "300": 2210,
+                                "32000": 2210,
+                                "400": 2210,
+                                "4095": 2210,
+                                "48000": 2210,
+                                "511": 2210,
+                                "64000": 2210,
+                                "8291": 2210
```

Comment:
Same here, if these fields are not fixed sizes we will have to think in other ways to store them to avoid fields explossion.

Reply:
Yes, we have fixed size buckets, see above!